### PR TITLE
Currency conversion

### PR DESCRIPTION
**Pull request by Musak2**
### Issue Id you have worked upon - 
#81

### Briefly explain your program logic - 
The MainCurrencyConverter React component is a simple currency converter application. It utilizes React hooks and Axios for API requests to provide currency conversion functionality. Here's a concise summary:

1. State Initialization: State variables for input amount, source (from) and target (to) currencies, options for dropdowns, and the converted output are initialized.

2. Data Fetching: When the source currency (from) changes, it fetches currency conversion rates from an API using Axios and stores the data in the info state.

3. Dropdown Options: Dropdown options are populated with available currencies based on the fetched data.

4. Conversion Logic: The convert function calculates the converted amount based on user input and the conversion rate.

5. Currency Switch: The flip function allows users to switch between source and target currencies.

6. User Interface: The component renders UI elements including input fields, dropdowns for currency selection, a switch button, and a "Convert" button to display the converted amount.

7. Styling the web application with CSS.

In summary, this component enables users to input an amount, select source and target currencies, and obtain the converted amount using live currency conversion rates from an API. They can also switch currencies for convenience.

### Screenshots(Attach 2 screenshots of your own input and output)

image

image

<br>

### By raising this PR I affirm that -
- [x] My code follows the guidelines of this project.<br>
- [x] I have performed a self-review of my own code.<br>
- [x] I have commented my code.<br>
- [x] My code gives the correct output.<br>

- [x] I affirm that I strictly follow contributing guidelines and code of conduct.

### CODE REVIEW

**Comment by SarthakKeshari** [comment]
@Musak2, 
As MaterialUI is the primary UI library kindly use - import Select from '@mui/material/Select'; for dropdown instead of import Dropdown from 'react-dropdown';
Similary for icon, kindly go for import CurrencyExchangeIcon from '@mui/icons-material/CurrencyExchange'; instead of import { HiSwitchHorizontal } from 'react-icons/hi';

Also remove the following dependencies from package.json -
    "react-dropdown": "^1.11.0",
    "react-icons": "^4.11.0",

**Comment by SarthakKeshari** [comment]
I apologize for the delay in my response, and I appreciate your understanding.

**Comment by Musak2** [comment]
I kindly announce that I have made corrections to the code as requested above. I also committed and pushed the code.

**Comment by SarthakKeshari** [comment]
Congratulations 🎉 @Musak2 on your contribution.
Thanks for adding value to this repository. 😊


Let's celebrate this fest 😃 by providing more developers 👨‍💻👩‍💻 an opportunity to contribute. Help us make this repository reach far and more by sharing this to your friends and colleagues. 
####  Your efforts can make difference.

**Comment by SarthakKeshari** [comment]
### Hey 👋, may you also encourage us by clicking the star icon on the repository main page. ⭐
image
